FirebaseAdmin/FirebaseAdmin/Auth/AuthErrorHandler.cs

```diff
@@ -63,6 +63,13 @@ internal sealed class AuthErrorHandler
                         AuthErrorCode.EmailAlreadyExists,
                         "The user with the provided email already exists")
                 },
+                {
+                    "EMAIL_NOT_FOUND",
```

Comment:
Also update the test case at: https://github.com/firebase/firebase-admin-dotnet/blob/e71b52a310b2e7bab461a51f4f3212cc7afa76d6/FirebaseAdmin/FirebaseAdmin.Tests/Auth/AuthErrorHandlerTest.cs#L27

Reply:
Done